### ISSUE TITLE
Sequence undroppable

### ISSUE THREAD

**Issue by borov333** [orientechnologies/orientdb-studio#403]
Hello support,

I've created a sequence called "accountId". Somehow it went into a state where it became impossible to drop it. Look at the commands below that verify the existence of the sequence and the error I'm getting when trying to drop it.

```
orientdb {db=Customers}> select sequence('accountId').current()
+----+--------+
|#   |sequence|
+----+--------+
|0   |4       |
+----+--------+
1 item(s) found. Query executed in 0.001 sec(s).
orientdb {db=Customers}> **drop sequence accountId**               

Error: com.orientechnologies.orient.core.exception.ORecordNotFoundException: **The record with id '#7:0' was not found**
	DB name="Customers"
	DB name="Customers"
orientdb {db=Customers}> select sequence('accountId').next()   
+----+--------+
|#   |sequence|
+----+--------+
|0   |5       |
+----+--------+

1 item(s) found. Query executed in 0.001 sec(s).
orientdb {db=Customers}> 
```

As we can see the sequence exists and working. I can clearly see that the sequence object does exist in the Database Management -> Structure tab under ID 7 as well, as shown in the attachment.

![database management](https://cloud.githubusercontent.com/assets/22382033/18769081/d5394a1e-80de-11e6-8ed7-b1219378782f.png)

I'm running OrientDB Community v 2.2.10. This seems to be an intermittent problem, because I've tried to reproduce it with other sequences and I was able to drop them easily. However, during my earlier tests I ran into a state when it occurred again.

**Comment by borov333** [comment]
I think I just replicated the issue. Seems that the **drop sequence** command is failing at all times. After I submitted issue #404 I went back to the console and attempted to drop the "testId" sequence. Console reported success, but my further check showed that the sequence was never dropped. See commands below:

orientdb {db=Customers}> **drop sequence testId**
Sequence removed successfully.
orientdb {db=Customers}> **select sequence('testId').next()**
+----+--------+
|#   |sequence|
+----+--------+
|0   |4       |
+----+--------+
1 item(s) found. Query executed in 0.002 sec(s).
orientdb {db=Customers}> **select sequence('testId').next()**
+----+--------+
|#   |sequence|
+----+--------+
|0   |5       |
+----+--------+
1 item(s) found. Query executed in 0.002 sec(s).
orientdb {db=Customers}> 

Then, I tried to drop the sequence second time and got an error, but I can still access it after the fact, see below:

orientdb {db=Customers}> **drop sequence testId**            
Error: com.orientechnologies.orient.core.exception.ORecordNotFoundException: The record with id '#7:3' was not found
	DB name="Customers"
	DB name="Customers"

orientdb {db=Customers}> **select sequence('testId').next()**
+----+--------+
|#   |sequence|
+----+--------+
|0   |6       |
+----+--------+
1 item(s) found. Query executed in 0.001 sec(s).
orientdb {db=Customers}>

**Comment by luigidellaquila** [comment]
Hi @borov333 

Thank you for reporting, I'll check it asap and let you know

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @borov333 

I just pushed a fix on 2.2.x, now I'm porting it to develop (3.0)

Thanks

Luigi

**Comment by borov333** [comment]
Thank you.
